<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraMotionSettingsWidget</class>
 <widget class="QWidget" name="CameraMotionSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="containerWidget">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QWidget" name="motionControlsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>16</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="motionDetectionCheckBox">
             <property name="text">
              <string>Motion Detection</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="motionStreamWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string comment="e.g. &quot;in Primary Stream&quot;">in</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="motionStreamComboBox"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="sensitivitySettingsWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="motionContainerWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>16</number>
              </property>
              <item>
               <widget class="QGroupBox" name="motionSensitivityGroupBox">
                <property name="title">
                 <string>Sensitivity</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="topMargin">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QPushButton" name="resetMotionRegionsButton">
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::MessageBar" name="motionHintBar" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="highResolutionAlertBar" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="motionImplicitlyDisabledAlertBar" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="recordingAlertBar" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="regionsAlertBar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::MessageBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::AlertBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
